docs/distributing-policies/secure-supply-chain.md

```diff
@@ -142,22 +163,35 @@ jobs:
         cosign sign ${IMMUTABLE_REF}
       # ... other build steps ...
 
-# ... remaining of the workflow file ...
+# ... remainder of the workflow file ...
 ```
-> **NOTE**: The policy templates repositories provided by the Kubewarden team
-> already have Github actions to build, test, sign and publish policies. It's
-highly recommended that policy developers use those templates.
 
-In the example above, we demonstrate using `kwctl` to publish the container image and find
-the reference to it. This is not mandatory. You can call `cosign` with the image
-reference generated elsewhere.
+</details>
+
+<!--TODO: In the note below, why is it highly recommended?-->
+:::note
 
+The Kubewarden policy templates already have Github actions to build, test, sign and publish policies.
+It's highly recommended that policy developers use those templates.
+
+:::
+
+<!--TODO:
+I need someone to explain what we are trying to say here.
+To be completely clear we should maybe reference the example more directly, say how we found the reference and what it was in the example, why isn't it mandatory and why might you want to do it. Where else and how would you get the image reference?
+-->
+In the example above, we demonstrated using `kwctl` to publish the container image and find the reference to it.
+This is not mandatory.
+You can call `cosign` with the image reference generated elsewhere.
```

Comment:
I think this is a leftover, we can remove it. It doesn't make any sense

Reply:
Thanks.